core/arch/arm/kernel/thread.c

```diff
@@ -234,7 +265,7 @@ void thread_unlock_global(void)
 }
 
 #ifdef ARM32
-uint32_t thread_get_exceptions(void)
+__noprof uint32_t thread_get_exceptions(void)
```

Comment:
I'd prefer to have the attributes after the type.

Reply:
ok